### PR TITLE
Add two new benchmarks

### PR DESCRIPTION
**Pull request by eeckstein**

### CODE REVIEW

**Comment by eeckstein** [comment]
@swift-ci Please smoke test and merge

File: benchmark/single-source/SortLargeExistentials.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+// A struct which exeeds the size of the existential inline buffer.
+struct Letter : LetterKind {
+  let value: String
+
+  var a: Int = 27
+  var b: Int = 27
+  var c: Int = 27
+  var d: Int = 27
```
Comment (palimondo) [inline benchmark/single-source/SortLargeExistentials.swift]: @eeckstein What’s the purpose of these four `Int`s that equal 27 in the Letter?

Comment (eeckstein) [inline benchmark/single-source/SortLargeExistentials.swift]: to make the struct larger than 3 words to ensure that it does not fit into the 3-word existential inline buffer.

Comment (palimondo) [inline benchmark/single-source/SortLargeExistentials.swift]: Gotcha! Could you add a comment that explains this?

Also [42](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life.2C_the_Universe.2C_and_Everything_.2842.29) is much better number… 👨‍💻💁‍♂️😇

Comment (eeckstein) [inline benchmark/single-source/SortLargeExistentials.swift]: https://github.com/apple/swift/pull/9088

**Pull request merged**